memory/src/main/scala/filodb.memory/format/vectors/Histogram.scala

```diff
@@ -324,6 +327,91 @@ final case class MaxHistogram(innerHist: MutableHistogram, max: Double) extends
   }
 }
 
+final case class DDSHistogram(relativeAccuracy: Double) extends HistogramWithBuckets {
```

Comment:
Let's add ScalaDoc here with description.... what are the units or bounds of relativeAccuracy?  what are typical values?